docs/docs/intro.md

```diff
@@ -4,32 +4,42 @@ slug: /
 
 # Welcome to MarbleRun
 
-MarbleRun is a framework for creating distributed confidential computing apps.
+MarbleRun is a framework for deploying distributed confidential computing applications.
+MarbleRun acts as a confidential operator for your deployment. Think of a trusted party in the control plane.
 
-Build your confidential microservices with [EGo, Gramine, or another runtime](features/runtimes.md), distribute them with Kubernetes on an SGX-enabled cluster, and let MarbleRun take care of the rest. Deploy end-to-end secure and verifiable AI pipelines or crunch on sensitive big data in the cloud. Confidential computing at scale has never been easier.
+Build your confidential microservices with tools like [EGo, Gramine, or similar runtime](./features/runtimes.md), orchestrate them with Kubernetes on an SGX-enabled cluster, and let MarbleRun take care of the rest.
+Deploy end-to-end secure and verifiable AI pipelines or crunch on sensitive big data in the cloud. Confidential computing at scale has never been easier.
 
-MarbleRun guarantees that the topology of your distributed app adheres to a manifest specified in simple JSON. MarbleRun verifies the integrity of services, bootstraps them, and sets up encrypted connections between them. If a node fails, MarbleRun will seamlessly substitute it with respect to the rules defined in the manifest.
+MarbleRun simplifies the process by handling much of the groundwork.
+It ensures that your app's topology adheres to your specified manifest.
+It verifies the identity and integrity of all your services, bootstraps them, and establishes secure, encrypted communication channels.
+As your app needs to scale, MarbleRun manages the addition of new instances, ensuring their secure verification.
 
-To keep things simple, MarbleRun issues one concise remote-attestation statement for your whole distributed app. This can be used by anyone to verify the integrity of your distributed app.
+MarbleRun provides comprehensive access and privilege management through its role-based access control (RBAC) feature. This can be used to enforce security policies such as 4-eye principles and other verification procedures in your operations.
+
+For added simplicity, MarbleRun offers a single remote attestation statement for your entire app. Anyone can use this to verify the integrity of your distributed app, making it easier to assure stakeholders of your app's security.
```

Comment:
```suggestion
To keep things simple, MarbleRun offers a single remote attestation statement for your entire app. Anyone can use this to verify the integrity of your distributed app, making it easier to assure stakeholders of your app's security.
```
The previously used phrase is much more common and is in line with the "Write like you speak" approach of the MS style guide.